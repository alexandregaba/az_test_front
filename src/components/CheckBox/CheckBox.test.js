import React from 'react';
import Checkbox from './Checkbox';
import { shallow } from 'enzyme';

const props = {
  checked: false,
  handleCheck: jest.fn(),
};

it('it triggers toggleCheckGroceryItem when itemCheck value is changed', () => {
  const Component = shallow(<Checkbox {...props} />);
  Component.find('input.checkbox').simulate('change', { target: { checked: true } });
  expect(props.handleCheck).toHaveBeenCalled();
});
