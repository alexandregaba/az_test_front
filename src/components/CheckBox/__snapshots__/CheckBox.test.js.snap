// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshots 1`] = `
<styled.div
  checked={false}
  className="check-box"
  onClick={[MockFunction]}
  type="checkbox"
/>
`;
